testsuite/integration/microprofile-tck/pom.xml

```diff
@@ -144,5 +280,87 @@
+                                    <goal>package</goal>
+                                </goals>
+                                <phase>${ts.bootable-jar-microprofile-tck-packaging.phase}</phase>
+                                <configuration>
+                                    <output-file-name>test-wildfly-microprofile-tck.jar</output-file-name>
+                                    <hollowJar>true</hollowJar>
+                                    <record-state>false</record-state>
+                                    <log-time>${galleon.log.time}</log-time>
+                                    <offline>true</offline>
+                                    <plugin-options>
+                                        <jboss-maven-dist/>
+                                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                    </plugin-options>
+                                    <feature-packs>
+                                        <feature-pack>
+                                            <!-- Child modules can set properties to override what feature pack is used -->
+                                            <groupId>${ts.microprofile-tck-provisioning.fp.groupId}</groupId>
+                                            <artifactId>${ts.microprofile-tck-provisioning.fp.artifactId}</artifactId>
+                                            <version>${ts.microprofile-tck-provisioning.fp.version}</version>
+                                        </feature-pack>
+                                    </feature-packs>
+                                    <layers>
+                                        <!-- Child modules would set properties to drive the desired layers -->
+                                        <layer>${ts.microprofile-tck-provisioning.base.layer}</layer>
+                                        <layer>${ts.microprofile-tck-provisioning.decorator.layer}</layer>
+                                    </layers>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <!-- Tests against the microprofile-tck bootable JAR  -->
+                        <configuration>
+                            <systemPropertyVariables>
+                                <install.dir>${project.build.directory}/wildfly</install.dir>
+                                <bootable.jar>${project.build.directory}/test-wildfly-microprofile-tck.jar</bootable.jar>
+                                <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                            </systemPropertyVariables>
+                            <redirectTestOutputToFile>${testLogToFile}</redirectTestOutputToFile>
```

Comment:
It's here because I just copied the config from child modules where it was present. You're right, it just duplicates what's already in the testsuite/integration/pom.xml and can be removed.